### ISSUE TITLE
Public Docker image no longer building

### ISSUE THREAD

**Issue by pataquets** [anacrolix/confluence#40]
[anacrolix/confluence - Docker Image on Docker Hub](https://hub.docker.com/r/anacrolix/confluence) does not have any published images. Maybe there's something broken (or disabled b/c failures?).
@anacrolix: Would you be interested in setting up this as it was done in anacrolix/dms#97?

**Comment by anacrolix** [comment]
Yes, would you be interested in maintaining it? You can do it the same way as anacrolix/dms, or I can link to something that you publish yourself as an official Docker image. Let me know what you need.
